ui-lib/src/androidTest/java/co/electriccoin/zcash/ui/screen/restore/view/RestoreViewTest.kt

```diff
@@ -98,6 +100,43 @@ class RestoreViewTest {
         }
     }
 
+    @Test
+    @MediumTest
+    fun automatic_selection_test() {
```

Comment:
I think this test case can be moved to `ParseResultTest` (and modified to not need Compose).

My thought process is we probably don't need to test at the UI layer if the underlying helper is correct, which then makes the tests faster and easier to maintain.  What do you think?

Reply:
Ok, agree. I'll move it.